src/Tests/Microsoft.DotNet.CommandFactory.Tests/GivenAProjectDependencyCommandResolver.cs

```diff
@@ -49,7 +49,7 @@ public void ItReturnsACommandSpecWhenToolIsInAProjectRef()
                 CommandName = "dotnet-portable",
                 Configuration = _configuration,
                 ProjectDirectory = testAsset.Path,
-                Framework = NuGet.Frameworks.FrameworkConstants.CommonFrameworks.NetCoreApp30
+                Framework = NuGet.Frameworks.FrameworkConstants.CommonFrameworks.Net60
```

Comment:
This would need to be updated each time we changed what the CurrentTargetFramework was.  Try the following:

```suggestion
                Framework = NuGet.Frameworks.NuGetFramework.Parse(ToolsetInfo.CurrentTargetFramework)
```

Likewise for the other changes in this file.